src/main/java/seedu/address/model/deck/Card.java

```diff
@@ -2,15 +2,26 @@
 
 import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
 
+import java.time.LocalDateTime;
 import java.util.Objects;
 
 /**
  * Represents a Card inside a Deck.
  */
 public class Card {
 
+    public static final double PERFORMANCE_MODERATING_FACTOR = -0.8;
```

Comment:
It would be nice if you can reference to the algorithms being used here.

Reply:
Alright, I'll add some documentation about the scheduling procedure shortly